dlk/python/dlk/plugins/tf.py

```diff
@@ -203,7 +203,7 @@ def attribute(self, attr_name: str) -> Any:
             attrs_data.append(self.nd_.attr[attr_name].list.i)
         elif attr_name in ['epsilon', 'alpha']:
             attrs_data.append(self.nd_.attr[attr_name].f)
-        elif attr_name == 'is_training' or attr_name == 'use_cudnn_on_gpu':
+        elif attr_name in ['is_training', 'use_cudnn_on_gpu', 'keep_dims']:
```

Comment:
Use tuple here

Reply:
Thanks for the spot, I will use tuple instead.